editor/inspector/input_event_editor_plugin.cpp

```diff
@@ -51,8 +52,57 @@ void InputEventConfigContainer::_event_changed() {
 
 void InputEventConfigContainer::_config_dialog_confirmed() {
 	Ref<InputEvent> ie = config_dialog->get_event();
-	input_event->copy_from(ie);
```

Comment:
Is it possible to make undo/redo action for this instead? I would simplify the code.